code/tasks/GetFreshdeskConfigTask.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+class GetFreshdeskConfigTask extends BuildTask
+{
+    public function run($request)
+    {
+        $ticketFields = [];
+        $agents = [];
+        $freshdesk = \FreshdeskAPI::create();
+
+        $result = $freshdesk->APICall('GET', FRESHDESK_API_BASEURL, '/api/v2/ticket_fields?type=default_status');
+
+        if ($result && $result->getStatusCode() == '200') {
+            $ticketFields = json_decode($result->getBody()->getContents(), true);
+
+            foreach ($ticketFields as $ticketField) {
+                $this->doStatuses($ticketField['choices']);
+            }
+        }
+
+        $result = $freshdesk->APICall('GET', FRESHDESK_API_BASEURL, '/api/v2/agents');
```

Comment:
Can you give `$result` less generic names here and on line 11? this is confusing as it is redeclared after beinh used @McJoppy